lib/output.js

```diff
@@ -67,7 +67,8 @@ function OutputStream(options) {
         screw_ie8        : false,
         preamble         : null,
         quote_style      : 0,
-        keep_quoted_props: false
+        keep_quoted_props: false,
+        ecma             : 5
```

Comment:
Hmmm, I prefer to focus to get the basic features working... Yeah I know, I personally don't want to work on it now, and others want so many features in. If there is too much to do I have to make choices and I got my eyes more focussed on the test262 tests now.